LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014-2024 The GoCryptoTrader Developers
+Copyright (c) 2014-2025 The GoCryptoTrader Developers
```

Comment:
```suggestion
Copyright 2014 The GoCryptoTrader Developers
```
As per google, microsoft and linux. Lots of citations and links [here](https://softwareengineering.stackexchange.com/questions/210472/is-renewal-of-mit-license-needed-on-github-at-the-beginning-of-each-year)

Reply:
I don't mind having a single year. Will get more perspective on it. @thrasher- @gloriousCode

Reply:
You can leave this as is, additional context here https://opensource.stackexchange.com/questions/6389/how-do-the-years-specified-in-a-copyright-statement-work

BTC devs do the same thing https://github.com/bitcoin/bitcoin/blob/master/COPYING#L3 but isn't a big deal at the end of the day

Reply:
Sorry; My link was incorrect, in that it didn't jump to the desired answer, and actually it's the top link that I found most compelling.

Citations and links are [here](https://daniel.haxx.se/blog/2023/01/08/copyright-without-years/) and [here](https://hynek.me/til/copyright-years/).

Copyright isn't licensing, it's actually implicit anyway under the Berne convention.

[Golang](https://github.com/golang/go/blob/master/LICENSE) was one of the most compelling arguments for me.

Also as per your link, which I kinda feel makes my point too:
> Nowadays copyright notices and the year in the copyright notice are no longer very important, since copyright starts automatically when a creative work is created

Resolving conversation; Just wanted to articulate my point correctly.

Reply:
It's fine, the link I provided shows all the necessary context anyway and thus my "Isn't a big deal" comment. So it can be left as is